### PR TITLE
IpInfo added outputs to the IP command

### PR DESCRIPTION
**Pull request by moishce**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/31377

## Description
IpInfo added outputs to the IP command

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by ShahafBenYakir — COMMENTED** [review]
I see that for the context standards we deviated a bit from the original design for example:
if we don't have `o.asn` then `ASN=o.org` or that `Organization` is an object containing `name` and `type` keys. It is ok but I want to make sure this is known to everyone, so can you summarize all these differences from the original design and write them down in the IpInfo issue and ask for Yael's opinion?

File: Packs/ipinfo/Integrations/integration-Ipinfo.yml [hunk 1]
```diff
@@ -80,20 +108,73 @@ script:
...
+      type: string
     - contextPath: IP.Geo.Country
       description: The IP country
+      type: string
     - contextPath: IP.Geo.Description
       description: The IP location as <City, Region, Postal Code, Country>
-    description: Check IP reputation (when information is available, returns a JSON with details).  Uses all configured Threat Intelligence feeds
+      type: string
+    - contextPath: IP.ASOwner
+      description: The IP ASN owner
```
Comment (ytauman) [inline Packs/ipinfo/Integrations/integration-Ipinfo.yml:132]: The IP AS owner (no N)

File: Packs/ipinfo/Integrations/integration-Ipinfo.yml [hunk 2]
```diff
@@ -80,20 +108,73 @@ script:
...
+      type: string
+    - contextPath: IP.FeedRelatedIndicators
+      description: Related Indicators
+      type: string
+    - contextPath: IP.Tags
+      description: the tags
```
Comment (ytauman) [inline Packs/ipinfo/Integrations/integration-Ipinfo.yml:138]: Change to 'Tags that are associated with the IP'

File: Packs/ipinfo/Integrations/integration-Ipinfo.yml [hunk 3]
```diff
@@ -80,20 +108,73 @@ script:
...
+      type: string
+    - contextPath: IP.Registrar.Abuse.Address
+      description: The Abuse Address
```
Comment (ytauman) [inline Packs/ipinfo/Integrations/integration-Ipinfo.yml:141]: remove title case + add suffix 'of the IP'

File: Packs/ipinfo/Integrations/integration-Ipinfo.yml [hunk 4]
```diff
@@ -80,20 +108,73 @@ script:
...
+      type: string
+    - contextPath: IP.Registrar.Abuse.Country
+      description: The Abuse Country
```
Comment (ytauman) [inline Packs/ipinfo/Integrations/integration-Ipinfo.yml:144]: remove title case + add suffix 'of the IP'

File: Packs/ipinfo/Integrations/integration-Ipinfo.yml [hunk 5]
```diff
@@ -80,20 +108,73 @@ script:
...
+      type: string
+    - contextPath: IP.Registrar.Abuse.Name
+      description: The Abuse Name
```
Comment (ytauman) [inline Packs/ipinfo/Integrations/integration-Ipinfo.yml:147]: remove title case + add suffix 'of the IP'

File: Packs/ipinfo/Integrations/integration-Ipinfo.yml [hunk 6]
```diff
@@ -80,20 +108,73 @@ script:
...
+      type: string
+    - contextPath: IP.Registrar.Abuse.Network
+      description: The Abuse Network
```
Comment (ytauman) [inline Packs/ipinfo/Integrations/integration-Ipinfo.yml:150]: remove title case + add suffix 'of the IP'

File: Packs/ipinfo/Integrations/integration-Ipinfo.yml [hunk 7]
```diff
@@ -80,20 +108,73 @@ script:
...
+      type: string
+    - contextPath: IP.Registrar.Abuse.Phone
+      description: The Abuse Phone
```
Comment (ytauman) [inline Packs/ipinfo/Integrations/integration-Ipinfo.yml:153]: remove title case + add suffix 'of the IP'

File: Packs/ipinfo/Integrations/integration-Ipinfo.yml [hunk 8]
```diff
@@ -80,20 +108,73 @@ script:
...
+      type: string
+    - contextPath: IP.Registrar.Abuse.Email
```
Comment (ytauman) [inline Packs/ipinfo/Integrations/integration-Ipinfo.yml:155]: remove title case + add suffix 'of the IP'

File: Packs/ipinfo/Integrations/integration-Ipinfo.yml [hunk 9]
```diff
@@ -80,20 +108,73 @@ script:
...
+      description: The Abuse Email
+      type: string
+    - contextPath: IP.Organization.Name
+      description: The Organization Name
```
Comment (ytauman) [inline Packs/ipinfo/Integrations/integration-Ipinfo.yml:159]: remove title case + add suffix 'of the IP'

File: Packs/ipinfo/Integrations/integration-Ipinfo.yml [hunk 10]
```diff
@@ -80,20 +108,73 @@ script:
...
+      type: string
     - contextPath: IP.Geo.Description
       description: The IP location as <City, Region, Postal Code, Country>
-    description: Check IP reputation (when information is available, returns a JSON with details).  Uses all configured Threat Intelligence feeds
+      type: string
+    - contextPath: IP.ASOwner
+      description: The IP ASN owner
+      type: string
+    - contextPath: IP.FeedRelatedIndicators
+      description: Related Indicators
```
Comment (ytauman) [inline Packs/ipinfo/Integrations/integration-Ipinfo.yml:135]: Change to 'Indicators that are associated with the IP'

File: Packs/ipinfo/Integrations/integration-Ipinfo.yml [hunk 11]
```diff
@@ -80,20 +108,73 @@ script:
...
+      description: The Abuse Phone
+      type: string
+    - contextPath: IP.Registrar.Abuse.Email
+      description: The Abuse Email
+      type: string
+    - contextPath: IP.Organization.Name
+      description: The Organization Name
+      type: string
+    - contextPath: IP.Organization.Type
+      description: The Organization Type
```
Comment (ytauman) [inline Packs/ipinfo/Integrations/integration-Ipinfo.yml:162]: remove title case + add suffix 'of the IP'

File: Packs/ipinfo/Integrations/integration-Ipinfo.yml [hunk 12]
```diff
@@ -80,20 +108,73 @@ script:
...
+      type: string
+    - contextPath: DBotScore.Indicator
```
Comment (ytauman) [inline Packs/ipinfo/Integrations/integration-Ipinfo.yml:164]: DbotScore objects aren't supposed to be directly associated with the indicator type (just talked about it the other day with aviad) - so please remove these.

**Comment by ShahafBenYakir** [comment]
Can we enhance the integration documentation here?

File: Packs/ipinfo/Integrations/README.md [hunk 13]
```diff
@@ -41,7 +41,22 @@ Check IP reputation (when information is available, returns a JSON with details)
...
+| IP.Registrar.Abuse.Name | string | The abuse name of the IP |
+| IP.Registrar.Abuse.Network | string | The abuse network of the IP |
+| IP.Registrar.Abuse.Phone | string | The abuse phone of the IP |
+| IP.Registrar.Abuse.Email | string | The abuse email of the IP |
+| IP.Organization.Name | string | description: The organization name of the IP |
+| IP.Organization.Type | string | description: The organization type of the IP |
+| DBotScore.Indicator | string | description: The indicator that was tested. |
+| DBotScore.Type | string | description: The indicator type. |
+| DBotScore.Vendor | string | description: The vendor used to calculate the score. |
+| DBotScore.Score | Number | description: The actual score. |
```
Comment (ShahafBenYakir) [inline Packs/ipinfo/Integrations/README.md:59]: We might a bug here. How did you add this to docs? manually or SDK? 
```suggestion
| IP.Organization.Name | string | The organization name of the IP |
| IP.Organization.Type | string | The organization type of the IP |
| DBotScore.Indicator | string | The indicator that was tested. |
| DBotScore.Type | string | The indicator type. |
| DBotScore.Vendor | string | The vendor used to calculate the score. |
| DBotScore.Score | Number | The actual score. |
```

File: Packs/ipinfo/ReleaseNotes/1_0_2.md [hunk 14]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### ipinfo
+- Added outputs to the IP command.
```
Comment (ShahafBenYakir) [inline Packs/ipinfo/ReleaseNotes/1_0_2.md:4]:
```suggestion
- Added new outputs to the IP command.
```

File: Packs/ipinfo/Integrations/README.md (hunk 13 above)
Comment (moishce) [reply Packs/ipinfo/Integrations/README.md:59]: manually

File: Packs/ipinfo/Integrations/integration-Ipinfo.yml [hunk 15]
```diff
@@ -80,20 +118,76 @@ script:
...
+      description: The abuse address of the IP
+      type: string
+    - contextPath: IP.Registrar.Abuse.Country
+      description: The abuse country of the IP
+      type: string
+    - contextPath: IP.Registrar.Abuse.Name
+      description: The abuse name of the IP
+      type: string
+    - contextPath: IP.Registrar.Abuse.Network
+      description: The abuse network of the IP
```
Comment (roysagi) [inline Packs/ipinfo/Integrations/integration-Ipinfo.yml:163]:
```suggestion
      description: The IP registrar abuse network
```

File: Packs/ipinfo/Integrations/integration-Ipinfo.yml [hunk 16]
```diff
@@ -80,20 +118,76 @@ script:
...
+      type: string
+    - contextPath: IP.Registrar.Abuse.Phone
+      description: The abuse phone of the IP
```
Comment (roysagi) [inline Packs/ipinfo/Integrations/integration-Ipinfo.yml:166]:
```suggestion
      description: The IP registrar abuse phone
```

File: Packs/ipinfo/Integrations/integration-Ipinfo.yml [hunk 17]
```diff
@@ -80,20 +118,76 @@ script:
...
+      type: string
+    - contextPath: IP.Registrar.Abuse.Email
+      description: The abuse email of the IP
```
Comment (roysagi) [inline Packs/ipinfo/Integrations/integration-Ipinfo.yml:169]:
```suggestion
      description: The IP registrar abuse email
```

File: Packs/ipinfo/Integrations/integration-Ipinfo.yml [hunk 18]
```diff
@@ -80,20 +118,76 @@ script:
...
+      type: string
+    - contextPath: IP.Organization.Name
+      description: The organization name of the IP
```
Comment (roysagi) [inline Packs/ipinfo/Integrations/integration-Ipinfo.yml:172]:
```suggestion
      description: The IP organization name
```

File: Packs/ipinfo/Integrations/integration-Ipinfo.yml [hunk 19]
```diff
@@ -80,20 +118,76 @@ script:
...
+      type: string
+    - contextPath: IP.Organization.Type
+      description: The organization type of the IP
```
Comment (roysagi) [inline Packs/ipinfo/Integrations/integration-Ipinfo.yml:175]:
```suggestion
      description: The IP organization type
```

**Review by roysagi — COMMENTED** [review]
Please see my comments and fix in the readme as well.

**Review by ShahafBenYakir — APPROVED** [review]
